src/main/java/solvd/ermakovich/ct/web/controller/VisitorController.java

```diff
@@ -0,0 +1,54 @@
+package solvd.ermakovich.ct.web.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import solvd.ermakovich.ct.domain.node.Visitor;
+import solvd.ermakovich.ct.repository.projection.PerformanceProjection;
+import solvd.ermakovich.ct.service.PerformanceService;
+import solvd.ermakovich.ct.service.VisitorService;
+import solvd.ermakovich.ct.web.dto.VisitorDto;
+import solvd.ermakovich.ct.web.mapper.VisitorMapper;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@RestController
+@RequestMapping("api/v1/visitors")
+@RequiredArgsConstructor
+public class VisitorController {
+
+    private final VisitorMapper mapper;
+    private final PerformanceService performanceService;
+    private final VisitorService visitorService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    VisitorDto create(@RequestBody VisitorDto visitorDto) {
```

Comment:
@kermakovich why methods are package-private?

Reply:
It's a mistake, but why should we use public methods? I can say only 1 reason: for tests. Are there any other reasons?

Reply:
https://stackoverflow.com/questions/38262666/springmvc-controller-public-or-default-protected I found this

Reply:
@kermakovich testing is the most important reason among others